doc/sphinx-guides/source/installation/config.rst

```diff
@@ -255,25 +255,23 @@ if your installation's :ref:`:PublicInstall` setting is true, or:
 
 You can configure this redirect properly in your cloud environment to generate a temporary URL for access to the Swift objects for computing.
 
-Amazon S3 Storage (or compatible)
+Amazon S3 Storage (or Compatible)
 +++++++++++++++++++++++++++++++++
 
-For institutions and organizations looking to use some kind of S3-based object storage for files uploaded to Dataverse,
-this is entirely possible. You can either use the services offered by Amazon or use some other, even on-site S3-compatible
-storage (like Minio, Ceph RADOS S3 Gateway and many more).
+For institutions and organizations looking to use Amazon's S3 cloud storage for their installation, this can be set up manually through creation of the credentials and config files or automatically via the AWS console commands. Alternatively, you may instead use other, even on-site S3-compatible storage (like Minio, Ceph RADOS S3 Gateway and many more).
```

Comment:
Sorry, but it is entirely possible to use the AWS utils for custom S3 services. Re-reading your changes, some people might get the impression that you can only use AWS CLI with Amazon's services.

IMHO using either AWS S3 or custom services are equal options, both absolutely valid. Of course, the different options might may work our differently for different people and deployment scenarios.

Reply:
Thanks @poikilotherm. @dlmurphy and I just talked about this. Good points. The challenge here is from a support perspective, as the team here at IQSS would be limited in the ability to diagnose and troubleshoot issues with custom services, but that would be better served by a specific message about support instead of presenting these custom options as a footnote. We'll revert this part of the document to your previous text.